docs/intrinsics/div128.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "_div128"
+ms.date: "03/27/2019"
+f1_keywords: ["_div128"]
+helpviewer_keywords: ["div128 intrinsic", "_div128 intrinsic"]
+ms.assetid: f68914b9-bffb-4e46-b1ba-4c249f7b4ecc
```

Comment:
`ms.assetid` and `ms.date` are copied from the corresponding multiplication intrinsic pages and will need to be updated. Not sure where these are generated or what they refer to however.

Reply:
The `ms.assetid` field isn't required for new content; it's a way of correlating the current content with the old MSDN content, which doesn't apply here. The `ms.date` field should be the current date, or something reasonably close. It's exposed on the topic page under the title, as something of a "last touched on" date.